models/chatMessage.go

```diff
@@ -14,3 +16,8 @@ type ChatMessage struct {
 func (s ChatMessage) String() string {
 	return s.Author + " says " + s.Body
 }
+
+//Valid checks to ensure the message is valid
+func (s ChatMessage) Valid() bool {
+	return s.Author != "" && s.Body != "" && s.ID != ""
```

Comment:
While using the rest api, I couldn't figure out what was wrong until I looked and seen that the `id` property was required. 😂  so that is why I opted to add this. One option we could do is use a library/package like https://github.com/teris-io/shortid for the id.

Reply:
I created an issue to keep a list of short-term todos for chat.  Server-generated IDs is the first on the list.  https://github.com/gabek/owncast/issues/26